Rakefile

```diff
@@ -140,12 +145,19 @@ namespace :version do
         downgrade_sql.puts("-- Downgrade SQL")
       end
       sh("git", "add", downgrade_sql_path)
+
+      if package_name == "pgroonga_database"
```

Comment:
Can we remove this `if`?

Reply:
fix: [0073cde](https://github.com/pgroonga/pgroonga/pull/744/commits/0073cdeca0e1a4b768a7f08b0b15710b627e250e) Yes, we can. This condition was needless.